packages/hashes.csv

```diff
@@ -1,42 +1,84 @@
+fetchai/skills/error/skill.yaml,QmaBu6AJmvQVXJDAiiBtLxufpjuBwWZ9zswGQr4x9MNTah
 fetchai/skills/error_test_skill,QmQ15evqAR2Dq9zyg7pszLm96tkihLxSjqrMycXvQFrTx3
+fetchai/skills/error_test_skill/skill.yaml,QmaXxkYRRD721SiovKgbBUEdmqx5LE6Lgs2S3AyMQQqpUB
 fetchai/skills/fipa_dummy_buyer,QmWGJgyfBFv5rNA4h9o71EPkFZxbdtB4rECSzguTKwMa1q
+fetchai/skills/fipa_dummy_buyer/skill.yaml,QmbhF4NPdkKmBvHE1Btf9QzWPrw4ezfESVCzf7KCEjscLE
 fetchai/skills/generic_buyer,QmdhDjDhHcCANKm8vKbbQnNuqazRYvbAUVRxNbcuhEe4mQ
+fetchai/skills/generic_buyer/skill.yaml,QmZfbsCHcbWccY22EJFgc46XM7qE7stB6w4QAvSLg2fTF4
 fetchai/skills/generic_seller,QmfL6YzD5VCHR15Q76od6zgcccJ3obHzhcPSH4fsKepeAT
+fetchai/skills/generic_seller/skill.yaml,QmfXYYTdmSgj9brrTHuAosBFENgb8kQidEg2gNBetTUEMn
 fetchai/skills/gym,QmTgQnnayFGWyhz4bQixTycqEVNwzWo8E8x8qKCX56hSGT
+fetchai/skills/gym/skill.yaml,QmcFroyJ8xZMRPXt7dFUr2DHWq8xYkQqhnri3W1jBSAZmL
 fetchai/skills/http_echo,QmTvhjUAwW9Ym3arJFfgjAcY6EerAZehKLmzBAeEo1cMNc
+fetchai/skills/http_echo/skill.yaml,QmeminTMAMwqMjuAuuiwcSU5K79WrC6HAZi7MDx2VZ55pX
 fetchai/skills/task_test_skill,QmYFTfauVZicfZum8vkDNYQCWdy8yvDMMqVtfNtXg741cu
+fetchai/skills/task_test_skill/skill.yaml,QmeEcvgSpMa921aVGZrjeKVNBnrj5SfDHGteMbkkgLEFj3
 open_aea/agents/gym_aea,QmUgCFwDyQJBxPDmyzxCKELNazpgkXCRQTaPLS3W5S34bt
+open_aea/agents/gym_aea/aea-config.yaml,QmQDmCQkmnTBBJGrsmsi2FvAQcKsQHX8FnvHuCNaUXzHyJ
 open_aea/agents/http_echo,QmcJewf19fn25iLdVZsXJMqpM9BQ17LBTS3TYpuy6XGUNL
+open_aea/agents/http_echo/aea-config.yaml,QmavyP2JqcrAK6EHvJTnPJZ3GAjPjSDigV868Pd12F4pNy
 open_aea/agents/my_first_aea,QmPZJPLDuiiXeKM1BsB3QRGiEoeqjS5hTXem5JAV2uiuSp
+open_aea/agents/my_first_aea/aea-config.yaml,QmRsNjTXsFVgdZg4LJyfwEAPDjCgJoLF2TnPJs6uXH2DNJ
 open_aea/connections/p2p_libp2p,QmYjjqzuLF6o4p7cHxakrDvKJYrWTScrqM3ui4WArj6dwQ
+open_aea/connections/p2p_libp2p/connection.yaml,QmWjzmT7f6JxJZm47WDVi7qj9KECChWcot4n2CBM96t63C
 open_aea/connections/p2p_libp2p_client,QmRcdrQ59dcn49pRvm4jPV1VC873VFNRBsFUkJUeBxqFCP
+open_aea/connections/p2p_libp2p_client/connection.yaml,QmYpi3m1RMSDVrvAUbnS52GTo63pfMApnVFCrmMyStT9eY
 open_aea/connections/p2p_libp2p_mailbox,QmXSJqE6gnHr5au2fUnNPSpBxv3N9Wyo6wNuzx2xytP7ar
+open_aea/connections/p2p_libp2p_mailbox/connection.yaml,QmWhZDA34ewPQroMQNiSqv5F9vcYtQhxDi2aWhFXJua3RQ
 open_aea/connections/scaffold,QmQeGqjcTvXn6sDGZCMUFTULjfHNW2RoCVN4PFeJUbbPWJ
+open_aea/connections/scaffold/connection.yaml,QmSNZgeHm1DRhhdzmsU3EKTTK4HyF4AU3dNEBaLFWF1XR7
 open_aea/contracts/scaffold,QmeqqfKY6pNLDbsCjBMxaNVdaHHd23NrpdHwYHhJ6DwqTB
+open_aea/contracts/scaffold/contract.yaml,QmZZ9g95RR9Yrbcp7ega7b4K8rFEs1r7Mp6Fr8qYwd5SZb
 open_aea/protocols/scaffold,QmSKEMPt7tC8pUukNnJzpMJrCTNxpgqVokgNBq1JToLB1t
+open_aea/protocols/scaffold/protocol.yaml,QmVJXv3czkzioSVmR3mqM58kdEEUbY9qc3xFw2V6PJbrrX
 open_aea/protocols/signing,QmXCoWUQjYLNFcybs5bpYJ8BTKtJK7KkyMWHsitdtcoLch
+open_aea/protocols/signing/protocol.yaml,QmeXkquWzNGF6qXnkz9TGsk3bmaS89CKTZHvuHTNHFWzzQ
```

Comment:
But there is a small issue with using a hash generated from IPFS daemon, it doesn't match with the hash generated using `IPFSHashOnly`.
```csv
open_aea/protocols/signing/protocol.yaml,QmSR9sDX5b1ZEQPHA2q997FEjSxYdzguZLhbnPazkPi4rg
fetchai/protocols/default/protocol.yaml,QmUU4hKwaheWn77dSV85352oyj3cApu53cBL3muAZDViqE
fetchai/skills/echo/skill.yaml,QmdhNmCyY3EcDrVrDAHvziKenjYXgvenGSqCtuHuT7FPeB
```

So should I use IPFSHashOnly in hashing script to generate hashes for *.yaml files?

Reply:
Not na issue anymore